src/handleSubmit.js

```diff
@@ -32,6 +35,9 @@ const handleSubmit = (submit, props, valid, asyncValidate, fields) => {
             return submitResult
           }, submitError => {
             const error = submitError instanceof SubmissionError ? submitError.errors : undefined
+            if (!error) {
+              setTimeout(() => throw submitError)
+            }
             stopSubmit(error)
```

Comment:
Yes, that's correct. It would work just as if there was an error in a plain javascript function.